pkg/cmd/operator/cmd.go

```diff
@@ -113,6 +120,11 @@ func NewOperator() *cobra.Command {
 			ctx, cancel := context.WithCancel(context.Background())
 			defer cancel()
 
```

Comment:
i would personally would have it seen like,

// create the file observer

// add reactor for proxy files

// run the file observer